ocp_resources/data_import_cron.py

```diff
@@ -57,8 +57,14 @@ def to_dict(self):
         super().to_dict()
         if not self.yaml_file:
             if self.image_stream and self.url:
-                raise ValueError("imageStream and url can not coexist")
-
+                raise ValueError("imageStream and url cannot coexist")
+            if not all(
+                [self.size, self.managed_data_source, self.pull_method, self.schedule]
```

Comment:
`self.managed_data_source` and `self.schedule` marked as required in the API docs but have defaults so we cannot force the user to send them.

Reply:
`self.size` is part of `storage` key which is not mandatory.

Reply:
Please checkout latest commits [approach](https://github.com/RedHatQE/openshift-python-wrapper/pull/1490/commits/2b9817657389968edf1c02c4a1de09c4252c4f2b#diff-bf49cfb5d402f8bb0e46e2125739760ec738f1a91689835bff8f8995f87beef1R99).
I can revert it in a second to the last usage of creating a storage on [L70](https://github.com/RedHatQE/openshift-python-wrapper/pull/1490/files#diff-bf49cfb5d402f8bb0e46e2125739760ec738f1a91689835bff8f8995f87beef1L70) and set also when `self.size` is None.

Reply:
Example output:
```
{'source': {'registry': {'pullMethod': '..', 'imageStream': '..'}}, 
'storage': {'resources': {'requests': {'storage': None}}, 'storageClassName': '..'}}
```